common/string_utils.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2020-2021 NVIDIA CORPORATION & AFFILIATES. ALL RIGHTS RESERVED.
+ *
+ * This software product is a proprietary product of Nvidia Corporation and its affiliates
+ * (the "Company") and all right, title, and interest in and to the software
+ * product, including all associated intellectual property rights, are and
+ * shall remain exclusively with the Company.
+ *
+ * This software product is governed by the End User License Agreement
+ * provided with the software product.
+ */
+
+#include <string>
+#include <vector>
+#include <list>
+#include <string>
+using namespace std;
```

Comment:
please remove using entire std namespace

use only what code below it really using

https://www.youtube.com/watch?v=etQX4Mme2f4